bin/dry-run.bash

```diff
@@ -21,13 +21,13 @@ if [[ $1 == "sc" ]]; then
   log_info "Running helmfile diff on the service cluster"
   : "${config[config_file_sc]:?Missing service cluster config file}"
 
-  "${here}/ops.bash" helmfile sc diff
+  "${here}/ops.bash" helmfile sc template | kubectl sc diff
```

Comment:
Could you give an example of what this solves?

Reply:
Actually not sure. @pettersv Do you have a good example?

Reply:
As noted in the issue `helm diff` does not check what's actually running in the cluster, but it was suggested that piping the template to `kubectl diff` could actually provide us with this. So if there was any manual changes we could see how that would impact this upgrade.

Reply:
Gotcha, I guess one should refrain from doing manual changes instead of going the route of adding things to git first etc, but I see the point and it's a good one!

Are we sure that doing `kubectl diff` instead of `helm diff` won't lose any information? It's hard to visualize, but could there be something that helm tracks that is not visible by just looking at the "plain k8s state"?

Reply:
Yes I think that everyone agrees that we should stay away from manual changes. But it will happen sometimes for different reasons.

At least I am not sure, so it would be very good to test (@robinelastisys have you been testing these changes, and if so then how have you tested it?). But I think you would get all relevant information.